lab_1/main.py

```diff
@@ -142,21 +127,18 @@ def detect_language(unknown_profile: dict, profile_1: dict,
     :return: a language
     """
 
-    if not isinstance(unknown_profile, dict):
-        return None
-    if not isinstance(profile_1, dict):
-        return None
-    if not isinstance(profile_2, dict):
-        return None
-    if not isinstance(top_n, int):
+    if not (isinstance(unknown_profile, dict)
+            and isinstance(profile_1, dict)
+            and isinstance(profile_2, dict)
+            and isinstance(top_n, int)):
         return None
     compare_1 = compare_profiles(unknown_profile, profile_1, top_n)
     compare_2 = compare_profiles(unknown_profile, profile_2, top_n)
     language = ''
     if compare_1 > compare_2:
-        language += profile_1["name"]
+        language = language.join(profile_1["name"])
```

Comment:
no, not good, lines are immutable, you can't change them
why can't you just write language = profile_1['name']?

Reply:
done

Reply:
No, you didn't get me. We'll discuss it today.